examples/mass_delete_old_medias.py

```diff
@@ -0,0 +1,460 @@
+#!/usr/bin/env python3
+"""
+Script which aims at freeing storage by deleting medias older than a
+given number of days. Depending on the parameters given, the script
+will either delete the medias or email speakers about the impending
+deletion, to give them time to protect their medias by applying a
+category to them.
+"""
+
+import argparse
+from datetime import date, datetime
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+import getpass
+import logging
+import os
+from pathlib import Path
+import smtplib
+import ssl
+import sys
+from typing import Optional
+
+try:
+    from ms_client.client import MediaServerClient
+except ModuleNotFoundError:
+    sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+    from ms_client.client import MediaServerClient
+
+
+logger = logging.getLogger(__name__)
+DEFAULT_PLAIN_EMAIL_TEMPLATE = (
+    'The following {media_count} media should be deleted according to '
```

Comment:
Add total size estimation

```suggestion
    'The following {media_count} media (total {media_size_pp}) should be deleted according to '
```